src/OptFrame/Evaluator.hpp

```diff
@@ -364,7 +364,7 @@ class Evaluator : public Direction
 
    // ============ betterThan ===========
 
-   using Direction::betterThan;
```

Comment:
This may break Evaluator class,  if Direction has multiple betterThan. Are you sure you can still do:
`Direction<XX>* d = new EvaluatorType<XX>` ?
It's likely `name hiding` will destroy this inheritance.